addons/snailmail_account/tests/test_snailmail_on_invoice.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.tests import TransactionCase
+
+
+class testAttachmentAccess(TransactionCase):
```

Comment:
the class name seems a little unrelated, maybe something like eg. `testSnailmailInvoice` would make more sense